mica/starcheck/tests/test_catalog_fetches.py

```diff
@@ -0,0 +1,86 @@
+             for i in range(0, 8)]
+    telem = fetch.MSIDset(['AOACASEQ', 'CORADMEN', 'AOPCADMD']
+                           + msids, date.secs, date.secs + 300)
+    att = fetch.MSIDset(['AOATTQT{}'.format(i) for i in [1, 2, 3, 4]], date.secs, date.secs + 300)
+    cat = {}
+    for slot in range(0, 8):
+        track = telem['AOACFCT{}'.format(slot)].vals == 'TRAK'
+        fid = telem['AOACFID{}'.format(slot)].vals == 'FID '
+        n = 30
+        if np.count_nonzero(track) < n:
+            continue
+        if np.any(fid & track):
+            cat[slot] = {'type': 'FID',
+                         'yag': telem['AOACYAN{}'.format(slot)].vals[fid & track][0],
+                         'zag': telem['AOACZAN{}'.format(slot)].vals[fid & track][0],}
+        else:
+            yags = []
+            zags = []
+            for sample in range(0, n):
+                qref = Quat(normalize([att['AOATTQT{}'.format(i)].vals[track][sample]
+                                       for i in [1, 2, 3, 4]]))
+                ra, dec = yagzag2radec(telem['AOACYAN{}'.format(slot)].vals[track][sample] / 3600.,
+                                       telem['AOACZAN{}'.format(slot)].vals[track][sample] / 3600.,
+                                       qref)
+                yag, zag = radec2yagzag(ra, dec, cmd_quat)
+                yags.append(yag)
+                zags.append(zag)
+            # This doesn't detect MON just yet
+            cat[slot] = {'type': 'STAR',
+                         'yag': np.median(yags) * 3600.,
+                         'zag': np.median(zags) * 3600.,}
+    return cat, telem
+
+
+def test_validate_catalogs_over_range():
+    start = '2017:001'
+    stop = '2017:005'
+    dwells = events.dwells.filter(start, stop)
+    for dwell in dwells:
+        telem_quat = get_cmd_quat(dwell.start)
+        cat, telem = get_trak_cat_from_telem(dwell.start, telem_quat)
+        sc = starcheck.get_starcheck_catalog_at_date(dwell.start)
+        sc_quat = Quat([sc['manvr'][-1]["target_Q{}".format(i)] for i in [1, 2, 3, 4]])
+        dq = sc_quat.dq(telem_quat)
+        if ((np.abs(dq.pitch) * 3600 > 1) or (np.abs(dq.yaw) * 3600 > 1)):
+            if dwell.manvr.template != 'unknown':
+                raise ValueError("Unexpected offset in pointing")
+            else:
+                print(dwell.start, "pointing offset but has unknown manvr template")
+                continue
+        trak_sc = sc['cat'][sc['cat']['type'] != 'ACQ']
+        for slot in range(0, 8):
+            if slot in cat:
+                slot_match = trak_sc[trak_sc['slot'] == slot]
+                if not len(slot_match):
+                    raise ValueError("missing slot in catalog")
+                trak_sc_slot = slot_match[0]
+                if trak_sc_slot['type'] == 'FID':
+                    assert cat[slot]['type'] == 'FID'
+                assert np.abs(cat[slot]['yag'] - trak_sc_slot['yang']) < 20.0
```

Comment:
It looks like this code that was doing the position checking needed a 20 arcsec limit because I was sharing the same limit between fids and stars, and the fids have "standard" offsets at this time.  I'll set different limits so this code captures that understanding.